3Dmol/WebGL/math/math.ts

```diff
@@ -1451,122 +1439,112 @@ export class Matrix3 {
-      this.direction = direction !== undefined ? direction : new Vector3();
-    }
-  
-    set(origin: Vector3, direction: Vector3) {
-      this.origin.copy(origin);
-      this.direction.copy(direction);
-  
-      return this;
-    }
-  
-    copy(ray: Ray) {
-      this.origin.copy(ray.origin);
-      this.direction.copy(ray.direction);
-  
-      return this;
-    }
-  
-    at(t: number, optionalTarget: Vector3) {
-      const result = optionalTarget || new Vector3();
-  
-      return result.copy(this.direction).multiplyScalar(t).add(this.origin);
-    }
-  
-    recast(t: any) {
-      const v1 = x;
-      this.origin.copy(this.at(t, v1));
-  
-      return this;
-    }
-  
-    closestPointToPoint(point: Vector3, optionalTarget: Vector3) {
-      const result = optionalTarget || new Vector3();
-      result.subVectors(point, this.origin);
-      const directionDistance = result.dot(this.direction);
-  
-      // returns a point on this ray
-      return result
-        .copy(this.direction)
-        .multiplyScalar(directionDistance)
-        .add(this.origin);
-    }
-  
-    distanceToPoint(point: Vector3) {
-      const v1 = x;
-      const directionDistance = v1
-        .subVectors(point, this.origin)
-        .dot(this.direction);
-      v1.copy(this.direction).multiplyScalar(directionDistance).add(this.origin);
-      return v1.distanceTo(point);
-    }
-  
-    // eslint-disable-next-line class-methods-use-this
-    isIntersectionCylinder() {}
-  
-    isIntersectionSphere(sphere: Sphere) {
-      return this.distanceToPoint(sphere.center) <= sphere.radius;
-    }
-  
-    isIntersectionPlane(plane: any) {
-      const denominator = plane.normal.dot(this.direction);
-  
-      // plane and ray are not perpendicular
-      if (denominator !== 0) return true;
-  
-      if (plane.distanceToPoint(this.origin) === 0) return true;
-  
-      return false;
-    }
-  
-    distanceToPlane(plane: any) {
-      const denominator = plane.normal.dot(this.direction);
-      if (denominator === 0) {
-        // line is coplanar
-        if (plane.distanceToPoint(this.origin) === 0) return 0;
-  
-        // ray is parallel
-        return undefined;
-      }
-  
-      const t = -(this.origin.dot(plane.normal) + plane.constant) / denominator;
-  
-      return t;
-    }
-  
-    intersectPlane(plane: any, optionalTarget: any) {
-      const t = this.distanceToPlane(plane);
-  
-      if (t === undefined) return undefined;
-  
-      return this.at(t, optionalTarget);
-    }
-  
-    applyMatrix4(matrix4: any) {
-      this.direction.add(this.origin).applyMatrix4(matrix4);
-      this.origin.applyMatrix4(matrix4);
-      this.direction.sub(this.origin);
-  
-      return this;
-    }
-  
-    equals(ray: {
-      origin: { equals: (arg0: any) => any };
-      direction: { equals: (arg0: any) => any };
-    }) {
-      return (
-        ray.origin.equals(this.origin) && ray.direction.equals(this.direction)
-      );
-    }
```

Comment:
this function needs to be removed because it was not defined in the original 3Dmol